docs/modules/general/pages/local-env.adoc

```diff
@@ -0,0 +1,26 @@
+= OneCX Local Environment
+
+== Overview
+With OneCX platform, every Application needs to adjusted so it can be integrated within the platform. For the best experience with Application development and integration, OneCX offers a https://github.com/onecx/onecx-local-env[OneCX Local Enviornment] repository. It provides a consistent setup of the platform and Core Applications, with minimal actions and allows extention.
```

Comment:
With **the** OneCX platform...
It provides a consistent setup of the platform and Core Applications with minimal actions and allows ~~extention~~ **extension**.